### DISCUSSION TITLE
Airtable - Relationship

### DISCUSSION THREAD

**Discussion by SDerrouazi** [Budibase/budibase#6449] [Help] [answered]
Hello,

Sorry if the question already exist, I didn't find any answer.

I use Airtable as a database with my Budibase application and in Airtable I have two tables has a relationship.

When I make a query in Budibase, I didn't find any solution to get this relationship provide by Airtable.

Do you have a solution ?

The idea for us is really to use budibase like a front-end and back-end solution but we would like to store our data in Airtable.

**Comment by melohagan** [comment, marked as answer]
Hello @SDerrouazi 

The Airtable API appears to be a fairly simple REST API. Thus to get data from relationship, you will need to make an additional API call:
https://community.airtable.com/t/how-to-get-linked-record-data-using-the-rest-api/26603

Sorry for the late reply, but it's only become possible to use the **Is in** filter with this latest release.
I've added an example to the docs here: https://docs.budibase.com/docs/airtable#retrieving-relational-data

**Reply by KenyOS** [reply]
I'm getting 404 on those links from the article you mentioned:
https://docs.budibase.com/docs/referencing
https://docs.budibase.com/docs/data-provider

**Reply by melohagan** [reply]
Thanks @KenyOS I've updated the docs page to use the correct link: https://docs.budibase.com/docs/chaining-actions

The data provider link appears to be working fine for me!
